gallery/tutorials/class_averaging.py

```diff
@@ -29,15 +29,15 @@
 # ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 L = 100
-round_disc = gaussian_2d(L, sigma_x=L / 4, sigma_y=L / 4)
+round_disc = gaussian_2d(L, sigma=(L / 4, L / 4))
```

Comment:
`sigma=L / 4`?

Reply:
Oops, thanks. I added the tuples before I added the scalar feature. Done.